server/src/pages/shot/text-tool.js

```diff
@@ -214,12 +214,19 @@ exports.TextTool = class TextTool extends React.Component {
     }
     const styles = window.getComputedStyle(this.textInput.current);
     const FONT_SIZE = parseInt(styles["font-size"], 10);
+    let line_height = styles["line-height"];
+
+    // Depending on the user agent computed styles
+    // line-height can have value as normal
+    if (line_height.toLowerCase() === "normal") {
+      line_height = 1.1 * FONT_SIZE;
```

Comment:
Using value 1.1[ (approximate default 1.2)](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#Values) to compute line-height for the fonts used in TextTool

Reply:
Hmm. This special case doesn't cover other weird stuff, like ems or percentages, that might be set as a user agent's default. Can we just set the line-height in the CSS directly?